setup.py

```diff
@@ -21,7 +21,7 @@
     ],
     packages=find_packages(exclude=["tests"]),
     install_requires=[
-        "django>=1.8,<1.9",
+        "django>=1.8,<2.0",
```

Comment:
Any reason why this is 2.0 instead of 1.12, like @thallada did here: https://github.com/edx/edx-proctoring/pull/361/files?

I honestly have no idea what's correct, but it seems like we ought to have them match.